products/zoningtaxlots/models/intermediate/int__zoningmapindex.sql

```diff
@@ -0,0 +1,83 @@
+{{ config(
+    materialized = 'table'
+) }}
+
+WITH dof_dtm AS (
+    SELECT * FROM {{ ref('stg__dof_dtm') }}
+),
+
+validdtm AS (
```

Comment:
I know this probably came from the source query, but lets refer to `int__valid_dtm` here

Reply:
@fvankrieken `int__valid_dtm` has an additional filter `WHERE ST_GEOMETRYTYPE(ST_MAKEVALID(geom)) = 'ST_MultiPolygon'` , which is slightly different from the `validdtm` here.  Should I check if that makes a difference?

Reply:
Yup - see if that filters the row count anymore. I imagine there are rows in dtm without geom, but those wouldn't be joined anyways, so it depends if there are any dtm rows with geom that are not multipolygon

Reply:
@fvankrieken `int__valid_dtm` does have one less row than `stg__dof_dtm`, seems that one row got filtered out by the multi-polygon restriction.

Reply:
Ooh. What is it?

Reply:
Oh actually it got filtered out for geom is null. I guess I can use `int__valid_dtm` then since the null won't get joined anyways? 

![Screenshot 2024-06-27 at 16 30 21](https://github.com/NYCPlanning/data-engineering/assets/148277458/a9937a05-a748-4e13-b5f2-da64aee24f3f)